jetstream/cli.py

```diff
@@ -191,3 +194,38 @@ def rerun_config_changed(project_id, dataset_id, dry_run):
     updated_external_configs = external_configs.updated_configs(project_id, dataset_id)
     for external_config in updated_external_configs:
         rerun(project_id, dataset_id, external_config.experimenter_slug, dry_run)
+
+
+@cli.command("validate_config")
+@click.argument("path")
```

Comment:
I think it's unixier to ask clients to write a command like `jetstream validate_config *.toml` instead of `jetstream validate_config .`.

Using nargs=-1 and forcing these to be of type click.Path would do that; both of those are covered in https://click.palletsprojects.com/en/7.x/arguments/